### PR TITLE
Conferencing

### PR DESCRIPTION
**Pull request by anatoliibugaiev**

### CODE REVIEW

**Comment by anatoliibugaiev** [comment]
Demo client for server-side conferencing

File: ringcentral-web-phone.iml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```
Comment (kirill-konshin) [inline ringcentral-web-phone.iml]: Don't commit IDEA files

**Comment by embbnux** [comment]
Any update in this PR?

**Comment by kirill-konshin** [comment]
Seems that the progress has stopped after initial review.

**Comment by kirill-konshin** [comment]
Are we going to do anything with it?

**Comment by vyshakhbabji** [comment]
There were multiple changes in API spec from the time we tested the conferencing API. You can close this PR as it needs further changes to the code.

**Comment by kirill-konshin** [comment]
Closing. Feel free to ping me once it's up to date.

**Pull request closed without merging**